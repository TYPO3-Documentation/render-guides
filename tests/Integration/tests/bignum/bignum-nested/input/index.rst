
==============================================
Nested bignums-xxl > bignums > Normally Styled
==============================================


..  rst-class:: bignums-xxl

1.  ONE One one bignums-xxl

    This is the story of my life ...

    ..  rst-class:: bignums

    1.  When I was young

        #.  this
        #.  and that
        #.  and this

    2.  When I was grown

        Oops, ...


    3.  When I was old

        Oh dear, ...



