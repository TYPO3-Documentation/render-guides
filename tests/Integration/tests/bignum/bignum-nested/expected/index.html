<!-- content start -->
                <section class="section" id="nested-bignums-xxl-bignums-normally-styled">
            <h1>Nested bignums-xxl &gt; bignums &gt; Normally Styled<a class="headerlink" href="#nested-bignums-xxl-bignums-normally-styled" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">&nbsp;<i class="fa-solid fa-link"></i></a></h1>


<ol class="bignums-xxl" start="1">
    <li value="1">
    <p>ONE One one bignums-xxl</p>

    <p>This is the story of my life ...</p>


<ol class="bignums" start="1">
    <li value="1">
    <p>When I was young</p>


<ol type="1">
    <li>this</li>
    <li>and that</li>
    <li>and this</li>
</ol>
</li>
    <li value="2">
    <p>When I was grown</p>

    <p>Oops, ...</p>
</li>
    <li value="3">
    <p>When I was old</p>

    <p>Oh dear, ...</p>
</li>
</ol>
</li>
</ol>

    </section>
        <!-- content end -->
